AndroidUnityWrapper~/unity/src/main/java/com/marigold/sdk/unity/EngageBySailthruWrapper.kt

```diff
@@ -181,6 +184,34 @@ object EngageBySailthruWrapper {
         })
     }
 
+    fun setAttributes(attributesString: String) {
```

Comment:
As discussed - take another look whether we can directly have a map as a parameter here.
Also check convention with the rest of the SDK. Don't want this to be the only place where we don't serialize.